### ISSUE TITLE
Invalid date message in search does not meet minimum contrast

### ISSUE THREAD

**Issue by joelvdavies** [ral-facilities/datagateway#907]
**Description**:
This occurs on search, in both light and dark modes on the search page.
![image](https://user-images.githubusercontent.com/90245114/140487202-81adf64d-dd0b-4e6e-99a3-5369ef519a60.png)
![image](https://user-images.githubusercontent.com/90245114/140487241-fcc0a61c-a9b0-4dae-9a56-d250d107f80a.png)

**Acceptance criteria**:
- Contrast issue fixed
